SubmitHGCalPGun.py

```diff
@@ -35,6 +35,8 @@ def parseOptions():
     parser.add_option('', '--rMax',  dest='rMax',  type=float, default=300.0,    help='max. r value')
     parser.add_option('', '--pointing',  action='store_false',  dest='pointing',  default=True,    help='pointing to (0,0,0) in case of closeby gun')
     parser.add_option('', '--overlapping',  action='store_true',  dest='overlapping',  default=False,    help='particles will be generated in window [phiMin,phiMax], [rMin,rMax] (true) or with a DeltaPhi=Delta/R (default false) in case of closeby gun')
+    parser.add_option('', '--randomshoot',  action='store_true',  dest='randomshoot',  default=False,    help='if true it will randomly choose one particle in the range [1, NParticles +1 ]')
```

Comment:
Can we use `randomShoot` rather than `randomshoot` to follow the camel case as used for the other options?